docs/advanced-topics/wes/wes-cwl-agc-tutorial.rst

```diff
@@ -0,0 +1,253 @@
+WES Tutorial for running CWL workflows through the Toil Engine on AWS AGC
+=========================================================================
+
+.. note::
+    Although the process for running WDL workflows through the Cromwell engine on AWS AGC is very similar to the process
+    described in this tutorial, there are a few differences. So, for information on how to launch a WDL workflow on AWS
```

Comment:
Nit and personal preference, I'd drop the `So, ` but either way.

Reply:
Fixed